tests/qunit/fixtures/media-widgets.js

```diff
@@ -0,0 +1,22 @@
+window._wpMediaImageWidget = {};
+window._wpMediaImageWidget.attachment = {
+	alt: '',
+	author: '1',
+	authorName: 'admin',
+	caption: '',
+	height: 1080,
+	id: 777,
+	link: 'http://src.wordpress-develop.dev/?attachment_id=777',
+	mime: 'image/jpeg',
+	name: 'Chicken and Ribs',
+	orientation: 'landscape',
+	sizes: {
+		medium: {
+			url: 'http://src.wordpress-develop.dev/wp-content/uploads/2017/04/chicken-and-ribs-300x300.jpg'
+		}
+	},
+	title: 'Chicken and Ribs',
+	type: 'image',
+	url: 'http://src.wordpress-develop.dev/wp-content/uploads/2017/04/chicken-and-ribs.jpg',
+	width: 1080
+};
```

Comment:
I don't think you necessarily need to store this in a separate fixture file. I think it's fine to just use the literal object in the call to `imageWidgetControlInstance.selectedAttachment.set()` inline.

In addition, it would be interesting to consider overriding/mocking the `jQuery.ajax` functionality for the `get-attachment` admin ajax request to return this data instead, so that you could do `control.model.set( { attachment_id: 777 } )` and then the mock would return the expected attachment to then populate it into `selectedAttachment`.